tests/test_workers/test_tasks/test_general.py

```diff
@@ -140,7 +140,6 @@ def test_create_bundle_archive(mock_gwc, mock_set_request, deps_present, tmpdir)
         expected |= set(deps_archive_contents.keys())
 
     assert bundle_contents == expected
-
-    mock_set_request.assert_called_once_with(
-        request_id, "in_progress", "Assembling the bundle archive"
-    )
+    call1 = (request_id, "in_progress", "Assembling the bundle archive")
+    call2 = (request_id, "complete", "Completed successfully")
+    mock_set_request.assert_has_calls(mock_set_request(call1), mock_set_request(call2))
```

Comment:
nitpick: should we also check mock_set_request was called twice (and only twice) here (previously, it was ensuring it was called only once)?

Reply:
@athos-ribeiro I didn't find out how to do that. Maybe add the same function call with any_order=True? I would appreciate any suggestion :)

Reply:
This is how we did it in the gomod tests:

https://github.com/release-engineering/cachito/blob/master/tests/test_workers/test_tasks/test_gomod.py#L66-L67

call_count docs: https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.call_count

Reply:
@ejegrova looks like mock objects have a `call_count` attribute: https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.call_count - should be what you need :)

Reply:
Thanks a lot. I changed that, is it ok like this? @athos-ribeiro